android/src/main/java/io/textile/rnmobile/TextileNode.java

```diff
@@ -680,6 +674,37 @@ public void run() {
         });
     }
 
+    @ReactMethod
+    public void searchContacts(final String query, final String options, final Promise promise) {
+        executor.execute(new Runnable() {
+            @Override
+            public void run() {
+                try {
+                    byte[] queryBytes = Base64.decode(query, Base64.DEFAULT);
+                    byte[] optionsBytes = Base64.decode(options, Base64.DEFAULT);
+                    node.searchContacts(queryBytes, optionsBytes, new Callback() {
```

Comment:
Looks like the Android version here still needs to be updated to work like iOS version